### PR TITLE
Create launchpad-social-ethcc.md

### PR DESCRIPTION
**Pull request by laeccles**
Please add this logo.

pl-launchpad

### CODE REVIEW

File: content/events/launchpad-social-ethcc.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+eventName: "Launchpad Social" #the name of your track or event and its mandatory
+
+eventDescription: "The Launchpad Social is a unique opportunity to create meaningful connections within the Protocol Labs Network. Embark on a floating location to experience a live pitch showcasing products & people tied to our ecosystem, and engage with current & past Launchpad participants, network companies & thought leaders in web3." #short description of your track or event limiting to 100-150 characters
+
+eventType: "Social" #Type of the event: Please choose one among the below options or just leave it blank
+
+location: "Paris, France" or "Location TBD" #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
```
Comment (WinsCodes) [inline content/events/launchpad-social-ethcc.md:8]: @laeccles Please provide just the required input if Paris as "Paris, France" or if the location is yet to be decided use "Location TBD"

File: content/events/launchpad-social-ethcc.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+eventName: "Launchpad Social" #the name of your track or event and its mandatory
```
Comment (WinsCodes) [inline content/events/launchpad-social-ethcc.md:2]: @laeccles Its always suggested better to remove comments wherever present (any content following #)

**Comment by laeccles** [comment]
@WinsCodes I am closing this PR until I receive confirmation regarding adding it.

File: content/events/launchpad-social-ethcc.md (hunk 2 above)
Comment (laeccles) [reply content/events/launchpad-social-ethcc.md]: @WinsCodes resolved

**Comment by WinsCodes** [comment]
@laeccles This event has been added using the Pull request: https://github.com/memser-spaceport/pln-events/pull/353

Closing this Pull request.

**Pull request closed without merging**